.gitignore

```diff
@@ -4,3 +4,4 @@ dev/
 pkg/
 Gemfile.lock
 .bundle/config
+.byebug_history
```

Comment:
Is this really something that should be part of the repository ?

Reply:
My bad, I realize this is the `.gitignore` so we don't really care. Will resolve this.